.github/CODEOWNERS

```diff
@@ -1 +1 @@
-@eliasvaltteri @henrinie @michalksiezuk-nc @jererobles @vastamaki @nikolauska @biphobe @ViktorHegyvari @aashirjaved @Nissinen @PerttuSavolainen @lintula @iriekun
+* @eliasvaltteri @henrinie @michalksiezuk-nc @jererobles @vastamaki @nikolauska @biphobe @PerttuSavolainen @lintula @iriekun
```

Comment:
Bohdan, Johanna and me are not in CODEOWNERS

Reply:
Thanks for pointing out, could you provide usernames so I could add the missing people?

Reply:
@Ben3eeE @BoRuDar 

It seems like Johanna is not in the nordcloud org yet?

Reply:
@terotil Could add Johanna to Nordcloud org?

Reply:
The username is @katjajohanna

Reply:
I've actually removed Johanna and Tero as we should only have the developers included in the list. You and Bogdan have been added to list with couple other changes included